lib/whitehall/search_index.rb

```diff
@@ -18,7 +28,7 @@ def self.indexer_class
     end
 
     def self.rummager_host
-      Plek.find('search')
+      Plek.find('rummager')
     end
 
     def self.add(instance)
```

Comment:
could we move this method before `self.for`? As I've read the comment and my mind is trying to find the `self.add`